response_operations_ui/templates/create-message.html

```diff
@@ -4,6 +4,8 @@
 
 {% block main %}
 
+<script src="/static/js/disable-send.js"></script>
```

Comment:
This isn't necessary as all JavaScript assets are minified into a single file and available to all templates that use the base template.

See: [minifier source](https://github.com/ONSdigital/response-operations-ui/blob/master/response_operations_ui/__init__.py#L22).
And: where the single file is included in [base.html](https://github.com/ONSdigital/response-operations-ui/blob/master/response_operations_ui/templates/layouts/base.html#L17).

Reply:
Removed it, forgot all the js got minified